### ISSUE TITLE
Creating SSH Keys

### ISSUE THREAD

**Issue by deleonl4** [susanBuck/e2-fall20#22]
Hi everyone,

My commands run after I input the email `ssh-keygen -t rsa -C “leslie.nicole.deleon@gmail.com` but I understand that it's not via SSH. It looks like this is a problem with Git? 
It's not recognizing the make a SSH key command. Any suggestions?

<img width="583" alt="Screen Shot 2020-09-17 at 11 26 57 AM" src="https://user-images.githubusercontent.com/47702390/93492556-ef526400-f8d8-11ea-88ec-7fe60b6e9ae3.png">

**Comment by susanBuck** [comment]
Hi @deleonl4 -

It looks like there may be a quote issue in the command you're running... note how the quote to the left of leslie is a curly quote `“` rather than a straight double quote `"`.

This can sometimes happen when copying/pasting text.

The full command should look like this:

```bash
ssh-keygen -t rsa -C "leslie.nicole.deleon@gmail.com"
```

**Not** this:
```bash
ssh-keygen -t rsa -C “leslie.nicole.deleon@gmail.com"
```

Give that a shot and let me know if it works.

**Comment by susanBuck** [comment]
Also, I noticed in your screenshot you were getting a `Command not found` error for some of the previous commands you ran. It looks like this happened because when you were copying and pasting commands from the course notes, you included the `$`. This isn't necessary, as the `$` is just there to signify the command prompt where you'll run the command.

Here's a screenshot to clarify that: 
![image](https://user-images.githubusercontent.com/237711/93493658-283f0880-f8da-11ea-879e-5a5f55507dd5.png)

This is a convention you'll see in a lot of documentation/tutorials etc. I realize now, though, it might not be obvious to those not familiar to it. Given this, I'm going to update the early command line notes with an explanation. Sorry for any confusion. : )

**Comment by deleonl4** [comment]
It's the simplest things. That's a great idea. Hopefully it will help future students who aren't familiar too!
Thank you so much!

**Comment by deleonl4** [comment]
One little thing (might be nothing at all) but wanted to make sure. After troubleshooting connecting the ssh key between my computer and github, I got it to work. However, I received the authenticity warning message below. Just wondering why it did this instead of immediately outputting "Hi (username), You've successfully authenticated, but GitHub does..."

<img width="713" alt="Screen Shot 2020-09-17 at 1 31 38 PM" src="https://user-images.githubusercontent.com/47702390/93506552-70662700-f8ea-11ea-994c-0e06ca8e0e72.png">

**Comment by susanBuck** [comment]
The authenticity warning is expected - it's your computer telling you you're connecting to a new host and asking you to confirm you trust that host. Once you say yes, it should no longer show that message again (for that given host).

**Comment by deleonl4** [comment]
Go it, I didn't see it on your video so I wanted to double check that I didn't connect with an unsafe host. Thank you again and again!

**Comment by deleonl4** [comment]
Not sure if I should open a new post; I am troubleshooting creating my repository. It failed the first time. I am trying to accomplish step 2 list below but I can't find the e2 directory. I went to the document root directory (local server's htdocs directory): `/Applications/XAMPP/xamppfiles/htdocs` 

<img width="1056" alt="Screen Shot 2020-09-17 at 2 07 25 PM" src="https://user-images.githubusercontent.com/47702390/93510448-32b7cd00-f8ef-11ea-98bd-6f5a181b2861.png">


**Repository Troubleshooting Steps:**
1. Follow the steps under the Delete section in your repo’s settings on Github.
2. Remove the e2 directory on your computer from your document root folder.

This was the error I received in the terminal:

<img width="651" alt="Screen Shot 2020-09-17 at 2 12 50 PM" src="https://user-images.githubusercontent.com/47702390/93510920-f5077400-f8ef-11ea-973d-29f34f4ed041.png">

Two potential solutions but I just downloaded xcode... and when attempting 1 I received the error below.

1- `$ xcode-select --install`
2- `$ sudo xcode-select --reset`

<img width="531" alt="Screen Shot 2020-09-17 at 2 16 51 PM" src="https://user-images.githubusercontent.com/47702390/93511303-968ec580-f8f0-11ea-9bd9-bc667cbb7f82.png">
